core/src/test/java/org/mule/el/datetime/DateTimeTimeTestCase.java

```diff
@@ -80,118 +82,112 @@ public void format()
     {
         SimpleDateFormat df = new SimpleDateFormat("hh:mm:ss");
         df.setTimeZone(TimeZone.getTimeZone("UTC"));
-        Assert.assertEquals(df.format(now.toDate()), now.format("hh:mm:ss"));
+        assertThat(now.format("hh:mm:ss"), is(df.format(now.toDate())));
```

Comment:
Compare against calendarNow instead of only now:

    assertThat(now.format("hh:mm:ss"), is(df.format(calendarNow.getTime())));